src/actions.jsx

```diff
@@ -1 +1,18 @@
 
+
+export const addTodo = (text) => ({
+  type: ADD_TODO,
+  payload: { text }
+});
+
+export const removeTodo = (id) => ({
+  type: REMOVE_TODO,
+  payload: { id }
+});
+
+export const updateTodo = (id, text) => ({
+  type: UPDATE_TODO,
+  payload: { id, text }
+});
```

Comment:
Where are the action types `ADD_TODO`,  `REMOVE_TODO`, `UPDATE_TODO ` coming from? They are not being imported or declared anywhere. Does the implementation work?